examples/mf8/mf8_regfile.prot

```diff
@@ -0,0 +1,33 @@
+struct Mf8RegFile {
+    in rd_addr: u5,
+    in rr_addr: u5, 
+    in write_en: u1,
+    in write_data: u8,
+
+    out rd_data: u8,
+    out rr_data: u8,
+    out z_reg: u16
+}
+
+// https://github.com/ekiwi/transactional-verification-with-protocols/blob/039fb8d5beafb3c4a464e5dac3dc8a3e00b69f04/enginev/check_mf8_reg.py#L28
+fn read_write<DUT: Mf8RegFile>(
```

Comment:
@ngernest : the description that you took this from does not have the same input persistence as our modern protocol language does. So you will have to add some explicit assignments with `X` to your translation.